Sources/Plasma/FeatureLib/pfGLPipeline/plGLEnumerate.cpp

```diff
@@ -261,6 +265,16 @@ void plCGLEnumerate(std::vector<hsG3DDeviceRecord>& records)
         hsG3DDeviceRecord devRec;
         devRec.SetG3DDeviceType(hsG3DDeviceSelector::kDevTypeOpenGL);
         devRec.SetDriverName("OpenGL.framework");
+        
+        plDisplayHelper* displayHelper = plDisplayHelper::CurrentDisplayHelper();
```

Comment:
@dpogue - FYI, I've upgraded the GL backend to hook into the new code.

Display helpers are a platform specific object that can vend display information to multiple backends. I kept this concept from the previous draft PR. So all rendering backends on the Mac can sit on the Mac display helper.

This lets OpenGL inherit all the behaviors I baked in for the Windows client like handling displays with a notch.